reana_client/cli/workflow.py

```diff
@@ -154,8 +154,7 @@ def workflow_workflows(ctx, _filter, output_format, access_token,
 
 @click.command(
     'create',
-    help='Create a REANA compatible workflow from REANA '
-         'specifications file.')
+    help='Create a REANA compatible workflow.')
```

Comment:
"Create a new analysis workflow..." since we are inside the REANA ecosystem, it is kind of clear that it will be REANA-compatible.

*(I can go through all the texts later...)*